Vagrantfile

```diff
@@ -48,7 +48,7 @@ Vagrant.configure("2") do |config|
   config.ssh.insert_key = false
 
   config.vm.box = "coreos-%s" % $update_channel
-  config.vm.box_version = ">= 308.0.1"
```

Comment:
We don't--I just figured 308.0.1 is ancient, and anyone using this `Vagrantfile` is probably looking to update to current stable at the least.